honeycomb-guide/src/project-structure/honeycomb-core.md

```diff
@@ -24,3 +25,8 @@ following:
 - **Orbit2**: Generic 2D implementation for orbit computation
 - **Vector2**: 2D vector representation
 - **Vertex2**: 2D vertex representation
+
+Note that optional features can be enabled when compiling this crate:
+
+- `utils` -- provides additionnal items that would qualify as quality-of-life, notably map generation functions.
+- `single_precision` -- uses `f32` instead of `f64` for coordinates representation in tests
```

Comment:
Do we need to expose that to the user?

Reply:
Not really, I'm removing this mention from the UG